packages/react-widgets/src/models/CategoryModel.js

```diff
@@ -1,87 +1,16 @@
-import { minify } from 'pgsql-minify';
-import { _filtersToSQL as filtersToSQL } from '@carto/react-core';
-import { executeSQL, SourceTypes } from '@carto/react-api';
 import { Methods, executeTask } from '@carto/react-workers';
 
 export const getCategories = async (props) => {
-  const {
-    data,
-    credentials,
-    column,
-    operation,
-    filters,
-    viewportFilter,
-    dataSource,
-    type,
-    opts,
-    alias = 'name'
-  } = props;
+  const { data, column, operationColumn, operation, filters, dataSource } = props;
 
   if (Array.isArray(data)) {
     throw new Error('Array is not a valid type to get categories');
   }
 
-  if (type === SourceTypes.BIGQUERY && !viewportFilter) {
-    throw new Error(
-      'Category Widget error: BigQuery layers need "viewportFilter" prop set to true.'
-    );
-  }
-
-  const operationColumn = props.operationColumn || column;
```

Comment:
@AdriSolid I think we must recover this line, otherwise operationColumn (optional) won't work on certain use cases